plugin/vim/quick-lint-js.vim/lua/lspconfig/quick_lint_js.lua

```diff
@@ -0,0 +1,30 @@
+local configs = require("lspconfig/configs")
+local util = require("lspconfig/util")
+
+configs.quick_lint_js = {
+    default_config = {
+        cmd = {"quick-lint-js", "--lsp-server"},
+        filetypes = {"javascript"},
+        root_dir = function(fname)
+            return util.find_node_modules_ancestor(fname) or
+                       util.find_git_ancestor(fname) or
+                       util.path.dirname(fname)
+        end
+    },
+    docs = {
+        description = [[
+https://quick-lint-js.com/
+quick-lint-js finds bugs in JavaScript programs.
+[200× faster than ESLint](https://quick-lint-js.com/benchmarks/),
+quick-lint-js gives you instant feedback as you code.
+Find bugs in your JavaScript before your finger leaves the keyboard.
+Lint any JavaScript file with no configuration.
+```sh
+npm install [-g] quick-lint-js
```

Comment:
I don't want to recommend that people install quick-lint-js through npm in order to get quick-lint-js working with their editor. Instead, people should use their system package manager (e.g. apt or Homebrew). I'd point people to the installation documentation instead: https://quick-lint-js.com/install/